### PR TITLE
Document how to contribute a new package to Host OS

### PR DESCRIPTION
**Pull request by lisibann**

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -205,3 +205,36 @@ Please see ``--help`` for more options.
 ::
 
 $ python host_os.py build-iso --help
+
+How to contribute a new package to Host OS
+------------------------------------------
+
```
Comment (alanoe) [inline README.rst]: As we talked personally today, simply point to documentation in the versions git repository from here and add this text there.

File: README.rst [hunk 2]
```diff
@@ -205,3 +205,36 @@ Please see ``--help`` for more options.
...
+Create a new directory at https://github.com/open-power-host-os/versions repository named <package_name>.
+
+Under this new directory: 
+
+1. Create a metadata file <package_name>.yaml. It will contain most of the information required to build the package. The package metadata (yaml) file format is at https://github.com/open-power-host-os/versions/package_metadata_yaml_format.yaml
```
Comment (alanoe) [inline README.rst]: metadata file => metadata file named

Comment (alanoe) [inline README.rst]: As we discussed today, I'll rename the file to package_metadata_yaml_template.yaml . Rename it here

File: README.rst [hunk 3]
```diff
@@ -205,3 +205,36 @@ Please see ``--help`` for more options.
...
+2. Create subdirectory "centOS/7", current "Linux Distribution Name"/"Linux Distribution Version", respectively
```
Comment (alanoe) [inline README.rst]: => Create two subdirectories named <distro_name> and <distro_name>/distro_version>, eg CentOS and CentOS/7

File: README.rst [hunk 4]
```diff
@@ -205,3 +205,36 @@ Please see ``--help`` for more options.
...
+3. Create rpm package specification file under <package_name>/centOS/7
```
Comment (alanoe) [inline README.rst]: => Create a rpm ackage specification file in <package_name>/<distro_name>/<distro_version>/<package_name>.spec

File: README.rst [hunk 5]
```diff
@@ -205,3 +205,36 @@ Please see ``--help`` for more options.
...
+4. Create (copy) the files required to build the package under <package_name>/centOS/7/SOURCES
```
Comment (alanoe) [inline README.rst]: => Copy the files required to build the package to <package_name>/<distro_name>/<distro_version>/SOURCES/

File: README.rst [hunk 6]
```diff
@@ -205,3 +205,36 @@ Please see ``--help`` for more options.
...
+ 
+When using %check stage at specification file, add macro to make possible to skip tests (check phase)
```
Comment (alanoe) [inline README.rst]: You are using "stage" and "phase" for the same concept, pls be consistent in terminology.

=> If the rpm specification file has a %check macro for running tests during rpm package build, add the following code to allow skipping these tests

File: README.rst [hunk 7]
```diff
@@ -205,3 +205,36 @@ Please see ``--help`` for more options.
...
+
+::
+
+   # The tests are disabled by default.
+   # Set --with tests or bcond_without to run tests.
+   # Original behaviour is preserved.
+   %bcond_with tests
+
+
+2. Add at the point of check stage:
```
Comment (alanoe) [inline README.rst]: stage => macro

**Review by alanoe — CHANGES_REQUESTED** [review]

**Comment by farosas** [comment]
I'd put this doc in the CONTRIBUTING.rst file. Regular usage of the repository does not require creating new packages.

File: README.rst (hunk 7 above)
Comment (lisibann) [reply README.rst]: Thanks Alan. I'll apply all these changes.

**Comment by lisibann** [comment]
I'll move documentation text to CONTRIBUTION.rst files on both builds and versions repositories.

**Comment by alanoe** [comment]
LA> I'll move documentation text to CONTRIBUTION.rst files on both builds and versions repositories.

Pls do not keep duplicated content on both files. The one in builds git repository should reference the one in versions git repository.

File: CONTRIBUTING.rst [hunk 8]
```diff
@@ -14,6 +14,9 @@ Also, please, follow the code style proposed by `PEP8
 <https://www.python.org/dev/peps/pep-0008/>`_. There is a tool called
 pycodestyle, formerly known as pep8, that checks if your code is compliant.
 
+For more details, please refer to `here
+<https://github.com/open-power-host-os/versions>`_.
```
Comment (ghost) [inline CONTRIBUTING.rst]: My understanding is that the `Contributing code` section refers to contributing with the Python scripts used in the builds.
I'd create a new section `Contributing packages` and write something like:
For details on enabling new packages to be built, refer to the `versions repository documentation <link to specific documentation>`_

**Comment by lisibann** [comment]
Yes. The builds/CONTRIBUTING.rst refers to versions.

File: CONTRIBUTING.rst (hunk 8 above)
Comment (lisibann) [reply CONTRIBUTING.rst]: Done.

File: CONTRIBUTING.rst [hunk 9]
```diff
@@ -14,6 +14,12 @@ Also, please, follow the code style proposed by `PEP8
 <https://www.python.org/dev/peps/pep-0008/>`_. There is a tool called
 pycodestyle, formerly known as pep8, that checks if your code is compliant.
 
+Contributing packages
+---------------------
+
+For details on adding new packages to the project, refer to the versions repository documentation `here
+<https://github.com/open-power-host-os/versions>`_.
```
Comment (ghost) [inline CONTRIBUTING.rst]: Place a link directly to the contributing page: `https://github.com/open-power-host-os/versions/blob/master/CONTRIBUTING.rst`

Comment (lisibann) [reply CONTRIBUTING.rst]: done

**Review by alanoe — APPROVED** [review]

**Review by ghost — APPROVED** [review]

**Pull request merged**